doc/overview.rst

```diff
@@ -97,18 +98,26 @@ Local optimization
 Common Name                                            Docs of the python class                                                                  Capabilities
 ====================================================== ========================================================================================= ===============
 Compass Search (CS)                                    :class:`pygmo.compass_search`                                                             S-CU
-COBYLA (from NLopt)                                    :class:`pygmo.nlopt`                                                                      S-CU
+COBYLA (from NLopt or SciPy)                           :class:`pygmo.nlopt`                                                                      S-CU
```

Comment:
For the algorithms common to both nlopt and scipy, can you make separate entries in the table? Like it is done now, if one clicks on the python class link, they get redirected to ``pygmo.nlopt``.

Reply:
Good point, addressed in 53a066f